main/parse.c

```diff
@@ -1158,6 +1189,28 @@ static void lazyInitialize (langType language)
 /*
 *   Option parsing
 */
+#define COUNT(D) (sizeof(D)/sizeof(D[0]))
+
+static void lang_def_flag_file_kind_long (const char* const optflag, const char* const param, void* data)
```

Comment:
This is really one of my bad. (Another one  is the usage of const.)

When I started working on fishman/ctags, I carelessly introduced many name with underscore.
> ~/var/ctags-github/main% grep _ *.c *.h | grep -v define | grep -v undef | grep -v TAG| grep -v ifdef | grep -v off_t | grep -v readtags  | grep -v endif| grep -v size_t | grep -v ifndef | grep -v '[A-Z]_[A-Z]' | wc -l
650

I introduced so many areas. I mixed the naming styles too many areas.
The unified style in all source code is better but I think it is too late to fix.